apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -26,6 +26,7 @@ defmodule AndiWeb.EditLiveView do
         <% [business] = inputs_for(f, :business) %>
         <% [technical] = inputs_for(f, :technical) %>
         <%= hidden_input(f, :id) %>
+        <%= hidden_input(f, :owner_id) %>
```

Comment:
This is fine, but now it's got me thinking of all the different ways someone could just hack in values for this, the dataset_id, etc. We'll A&C that